styling-sass-sample/styles.scss

```diff
@@ -0,0 +1,18 @@
+.SassStylingExtension {
+  .doodle-style {
+    width: 200px;
+    background-color: cadetblue;
+    padding: calc(var(--padding) * 2);
+
+    // Check if light theme enabled in Lens
+    .theme-light & {
```

Comment:
probably we should specify top-level theme element more precisely, like `body[data-theme="light"]`

Reply:
I'm working on this, changes will be in different PR.